.github/workflows/release-backend.yaml

```diff
@@ -0,0 +1,14 @@
+on:
+  push:
+    tags:
+      - ccdscan/*
+
+jobs:
+  release-base-image:
+    uses: concordium/.github/.github/workflows/docker-release-workflow.yaml@main
```

Comment:
We do not have to fix the entire organisations release flow using this generalisation. 

We have certain release flow, they seem much alike, lets ensure consistency. I do want us to come into a repository and know how we release in this repository. We have not been consistent in the past. 

Furthermore such generalisation would ease the use of shared components in the future such as smoke test etc.

Reply:
If you want content which differ:
prefix, version format, version validation, service name conventions.

And yes fixing secrets namings helps us. The next thing I am going to do is to automate this secrets generation from within terraform. Then it helps that we have fixed docker secrets.

Reply:
I would much rather have the release flow in this repo rather than code golfing this.
Consistency is not generalizing things, and this is introducing a new approach yet again, which makes this inconsistent with what we already have.